src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -35,17 +46,52 @@ class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+
+  override fun esInfluyente(): Boolean = false
 }
 
+
+
 // A este tipo de List no se le pueden agregar elementos una vez definida
 class Viajante(val provinciasHabilitadas: List<Provincia>) : Vendedor() {
+
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return provinciasHabilitadas.contains(ciudad.provincia)
   }
+
+  override fun esInfluyente(): Boolean = provinciasHabilitadas.sumBy {p -> p.poblacion} >= 10000000
 }
 
 class ComercioCorresponsal(val ciudades: List<Ciudad>) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudades.contains(ciudad)
   }
+  fun provincias() = ciudades.map { c -> c.provincia }.toSet()
+
+  override fun esInfluyente(): Boolean = (ciudades.size > 4) or (this.provincias().size >2)
+
+}
+
+class CentroDeDistrib(val ciudadActual: Ciudad) {
```

Comment:
No ahorres teclas, escribí el nombre completo.

```suggestion
class CentroDeDistribucion(val ciudadActual: Ciudad) {
```